recipe/meta.yaml

```diff
@@ -155,7 +155,7 @@ outputs:
         - async-timeout
         - colorful
         - gpustat >=1.0.0
-        - grpcio
+        - grpcio >=1.56,!=1.58
```

Comment:
May want to loosen

Reply:
yep good call, maybe we just do != to the version known to be bad. the down side is if there are more bad older versions, we'll need to repodata patch again...

Reply:
As per: https://github.com/conda-forge/conda-forge-repodata-patches-feedstock/pull/638#issuecomment-1900702894 I don't think `!=1.58` is sufficient.

The following incorrectly solves with grpcio 1.58.2:

```
---
name: test
channels:
  - nodefaults
  - conda-forge
dependencies:
  - python 3.11
  - ray-all 2.9.0
  - jax 0.4.23
  - grpcio !=1.58
```

The following will (correctly) fail to solve:
```
---
name: test
channels:
  - nodefaults
  - conda-forge
dependencies:
  - python 3.11
  - ray-all 2.9.0
  - jax 0.4.23
  - grpcio !=1.58.1, !=1.58.2
```

Reply:
updated, bugs abound https://github.com/conda/conda/issues/11744